script.js

```diff
@@ -0,0 +1,61 @@
+let collection = JSON.parse(localStorage.getItem('collection')) || [];
+const add = (title, author) => {
+  const book = { title, author };
+  collection.push(book);
+  localStorage.setItem('collection', JSON.stringify(collection));
+};
```

Comment:
- I think it would be better to favor the use of array methods that don't mutate the array, e.g. concat, filter, slice instead of push, pop, splice, and so on.